### PR TITLE
created new portfolio page with enhanced ui

### PR DESCRIPTION
**Pull request by Ro-shni**

### CODE REVIEW

**Comment by Ro-shni** [comment]
fixed issue#61  kindly suggest changes if required!

File: portfolio new/index.html [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+<link href="https://fonts.googleapis.com/css2?family=Montserrat&family=Nunito&family=Satisfy&display=swap" rel="stylesheet">
+  <!-- my css -->
+  <link rel="stylesheet" href="css/style.css"> 
+<!-- title -->
+    <title>Student | Portfolio</title>
+</head>
+<body>
+  <!-- navbar -->
+  <main-header>
+
```
Comment (0silverback0) [inline portfolio new/index.html]: add a link here in your home tab to /index.html for ease of navigation back to repo landing page

File: portfolio new/index.html
Comment (0silverback0) [inline portfolio new/index.html:1]: also in the projects root directory add your information to the index.html file so that your project can be merged

**Review by 0silverback0 — CHANGES_REQUESTED** [review]

**Comment by Ro-shni** [comment]
I've made the requested changes could you verify it and update me?

**Comment by 0silverback0** [comment]
image

Add your info in the root index.html so your project can be viewed

**Comment by Ro-shni** [comment]
image


I've updated it. could you please recheck?

**Comment by Ro-shni** [comment]
image

portfolio new  pr#62 issue #61

**Comment by 0silverback0** [comment]
please just pull the latest changes and commit then

**Comment by Ro-shni** [comment]
@0silverback0 , yes I have committed the latest pull. could u verify and add Hactoberfest accepted label to this pr?

**Pull request merged**